### PR TITLE
General Info proofreading

### PR DESCRIPTION
**Pull request by quinntracy**

### CODE REVIEW

File: content/en/docs/refguide/general/mx-command-line-tool/module.md [hunk 1]
```diff
@@ -113,14 +113,14 @@ If there are errors, the exit code consists of three digits: XYZ where:
 
 * Z indicates the error details:
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/general/mx-command-line-tool/module.md]: I think this is a copy-paste from the errors as they appear in the UI so we can leave them as is.

**Review by MariaShaposhnikova — COMMENTED** [review]
Hi Quinn, all looks good, I left only one comment.

File: content/en/docs/refguide/general/mx-command-line-tool/module.md (hunk 1 above)
Comment (quinntracy) [reply content/en/docs/refguide/general/mx-command-line-tool/module.md]: Got it, I'll revert these changes

Comment (quinntracy) [reply content/en/docs/refguide/general/mx-command-line-tool/module.md]: To confirm, if it's a copy-paste, should "can't" be left as is?

**Pull request merged**